pkgs/development/python-modules/pandas/default.nix

```diff
@@ -55,51 +62,47 @@ buildPythonPackage rec {
     xlwt
   ];
 
-  checkInputs = [ pytestCheckHook glibcLocales hypothesis ];
+  checkInputs = [
+    glibcLocales
+    hypothesis
+    jinja2
+    pytest-asyncio
+    pytest-xdist
```

Comment:
Adding xdist without enabling it (e.g. `-n auto`) in pytestFlagsArray won't do much.